packages/components/src/templates/quote.stories.ts

```diff
@@ -17,174 +17,103 @@ export default {
   }
 };
 
-const img = (hidden = false) => html`
-  <img
-    class="w-16 h-16 rounded-full overflow-hidden object-cover ${hidden ? 'hidden' : ''}"
-    src="./placeholders/images/family.jpg"
-    alt="A caring father, balancing one child on his shoulder and gently cradling another in his arms."
-  />
-`;
-
-const information = (hidden = false, text = '4xl') => (hidden ? '' : html`<p class="mt-4 text-sm">${text}</p>`);
-
-/**
- * The default version of a `quote` pattern showcases a quote in an appealing layout,
- * accompanied by an image of the speaker, along with their name and job description.
- * This adds context and personality to the quote, making it more engaging and relatable.
- */
-
-export const Default = {
-  render: () =>
-    html`<div class="text-left text-black">
-      <q class="block mb-6 text-3xl lg:text-4xl leading-tight"
-        >Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</q
-      >
-      <div class="gap-4 flex items-center">
-        ${img()}
-        <div>
-          <p class="sd-paragraph font-bold mb-1">Jane Miller</p>
-          <p class="sd-paragraph sd-paragraph--size-sm">Job Title</p>
-        </div>
-      </div>
-    </div>`
-};
-
-/**
- * This version of the quote provides flexibility in terms of `size` and offers three options: 4xl, 3xl, and xl.
```

Comment:
why did we remove all the information on the quote? as we don't have a component but only the template, I see a need for some further information on how to use it.